great_expectations_cloud/agent/cli.py

```diff
@@ -36,26 +38,43 @@ def _parse_args() -> Arguments:
         default=False,
         type=pathlib.Path,
     )
+    parser.add_argument(
+        "--json-log",
+        help="Output logs in JSON format. Defaults to False.",
+        default=False,
+    )
     parser.add_argument(
         "--log-cfg-file",
         help="Path to a logging configuration json file. Supersedes --log-level and --skip-log-file.",
         type=pathlib.Path,
     )
+    parser.add_argument(
+        "--environment",
+        help="The environment in which agent is running",
+        type=str,
+        default="production",
+    )
```

Comment:
I think this is confusing as a CLI argument since it doesn't do anything other than add this value to the log message.

Reply:
I agree it suggests behavior but does not

Reply:
removed `environment` and converted to custom tags in eb5de16a66b56f7c2cb8988172022b84a768b502